plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHiveLocal.java

```diff
@@ -126,4 +161,139 @@ public void testGetTableSchemaOffline()
     {
         throw new SkipException("Test disabled for this subclass");
     }
+
+    @Test
+    public void testSparkBucketedTableValidation()
+            throws Exception
+    {
+        SchemaTableName table = temporaryTable("spark_bucket_validation");
+        try {
+            doTestSparkBucketedTableValidation(table);
+        }
+        finally {
+            dropTable(table);
+        }
+    }
+
+    private void doTestSparkBucketedTableValidation(SchemaTableName tableName)
+            throws Exception
+    {
+        java.nio.file.Path externalLocation = extractResourceZipToTemporaryDirectory("spark_bucketed_nation.zip");
+        try {
+            createExternalTable(
+                    tableName,
+                    ORC,
+                    ImmutableList.of(
+                            new Column("nationkey", HIVE_INT, Optional.empty()),
+                            new Column("name", HIVE_STRING, Optional.empty()),
+                            new Column("regionkey", HIVE_INT, Optional.empty()),
+                            new Column("comment", HIVE_STRING, Optional.empty())),
+                    ImmutableList.of(),
+                    Optional.of(new HiveBucketProperty(
+                            ImmutableList.of("nationkey"),
+                            BUCKETING_V1,
+                            3,
+                            ImmutableList.of(new SortingColumn("name", SortingColumn.Order.ASCENDING)))),
+                    new Path(URI.create("file://" + externalLocation.toString())));
+
+            // read fails due to validation failure
```

Comment:
Why overridden ?

Reply:
What's overridden here? This is a new test with a different name, it doesn't override anything from super.